aspnetcore/grpc/test-tools.md

```diff
@@ -1,27 +1,29 @@
 ---
-title: Test gRPC services with gRPCurl in ASP.NET Core
+title: Test gRPC services with Postman or gRPCurl in ASP.NET Core
 author: jamesnk
-description: Learn how to test services with gRPC tools. gRPCurl a command-line tool for interacting with gRPC services. gRPCui is an interactive web UI.
+description: Learn how to test services with gRPC tools. Postman is an interactive web UI. gRPCurl a command-line tool for interacting with gRPC services. gRPCui is an interactive web UI.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: jamesnk
 ms.date: 03/31/2022
 uid: grpc/test-tools
 ---
-# Test gRPC services with gRPCurl in ASP.NET Core
+# Test gRPC services with Postman or gRPCurl in ASP.NET Core
 
 By [James Newton-King](https://twitter.com/jamesnk)
 :::moniker range=">= aspnetcore-6.0"
 Tooling is available for gRPC that allows developers to test services without building client apps:
 
-* [gRPCurl](https://github.com/fullstorydev/grpcurl) is a command-line tool that provides interaction with gRPC services.
-* [gRPCui](https://github.com/fullstorydev/grpcui) builds on top of gRPCurl and adds an interactive web UI for gRPC, similar to tools such as Postman and Swagger UI.
+* [Postman](https://www.postman.com/) is an API platform with an interactive UI for calling APIs. Postman can run in the browser or be downloaded and run locally. Postman supports calling gRPC services.
+* [gRPCurl](https://github.com/fullstorydev/grpcurl) is an open-source command-line tool that provides interaction with gRPC services.
+* [gRPCui](https://github.com/fullstorydev/grpcui) builds on top of gRPCurl and adds an open-source interactive web UI for gRPC.
 
 This article discusses how to:
 
 * Set up gRPC reflection with a gRPC ASP.NET Core app.
```

Comment:
```suggestion
* Set up gRPC server reflection with a gRPC ASP.NET Core app.
```

Reply:
Very minor item: Maybe everyone refers to this as just "gRPC reflection" and it should stay that way, but I'll suggest the more descriptive alternative.